tweets/migrations/0002_tweet_likes.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 5.0.3 on 2024-07-25 10:41
+
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tweets', '0001_initial'),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='tweet',
+            name='likes',
+            field=models.ManyToManyField(related_name='liked_by', to=settings.AUTH_USER_MODEL),
+        ),
+    ]
```

Comment:
Change this to create a Like Model with 

TweetId 
UserId

Reply:
migrations are automatically generated in django. I dont think modifying this will work.